community-members.md

```diff
@@ -0,0 +1,44 @@
+# OpenFeature Community Members
+
+## Governance Board
+
+This is the current Governance Committee, per the Governance Committee Charter, in alphabetical order:
+
+- [Alois Reitbauer](https://github.com/aloisreitbauer), Dynatrace, term: Oct 1st, 2022 - TBD
+- [Ben Rometsch](https://github.com/dabeeeenster), Flagsmith, term: April 28, 2022 - TBD
+- [Justin Abrahms](https://github.com/justinabrahms), eBay, term: April 28, 2022 - TBD
+- [Kevin Chu](https://github.com/kbychu), GitLab, term: April 28, 2022 - TBD
+- [Michael Beemer](https://github.com/beeme1mr), Dynatrace, term: April 28, 2022 - TBD
+- [Pete Hodgson](https://github.com/moredip), Independent, term: April 28, 2022 - TBD
+
+currently there is one vacant seat
+
+> NOTE:
+> In April 2022 _Project Maintainers_ assigned five seven individuals to be members of the _Bootstrap Governing Committee_.
+> This is a **temporary** arrangement that should be replaced by an elected governing body before March 2024.
+> Two seats remain vacant at the moment
+
+## Technical Commitee
+
+This is the current Technical Committee, per the Technical Committee Charter, in alphabetical order:
+
+- [Dan O’Brien](https://github.com/InTheCloudDan), LaunchDarkly, term: April 28, 2022 - April 28, 2023
+- [Todd Baert](https://github.com/toddbaert), Dynatrace, term: April 28, 2022 - April 28, 2023
+- [Steve Arch](https://github.com/agentgonzo), CloudBees, term: April 28, 2022 - April 28, 2023
+
+## Community Management
+
+## Specifications and Approvers
+
+Members of the Technical Committee are the maintainers of OpenFeature spec
+
+Spec Approvers:
+
+
+
+## Java SDK
```

Comment:
Is there a reason we have the Java SDK but none of the other 1.0 SDKs?

https://github.com/open-feature/js-sdk
https://github.com/open-feature/go-sdk
https://github.com/open-feature/dotnet-sdk

I've added the `sdk` topic **to all the sdk repos**, so you can just use this query to return all of them: https://github.com/search?q=org%3Aopen-feature+sdk+in%3Atopics

Reply:
because  I have not yet aded all of them. This is the next step to get all of them in.